operations/src/main/java/com/nextdoor/bender/operation/gelf/GelfOperation.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * Copyright 2018 Nextdoor.com, Inc
+ *
+ */
+
+package com.nextdoor.bender.operation.gelf;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Map.Entry;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.nextdoor.bender.InternalEvent;
+import com.nextdoor.bender.deserializer.DeserializedEvent;
+import com.nextdoor.bender.operation.Operation;
+import com.nextdoor.bender.operation.OperationException;
+import com.nextdoor.bender.operation.json.key.FlattenOperation;
+import com.nextdoor.bender.operation.substitution.SubstitutionOperation;
+import com.nextdoor.bender.operation.substitution.SubstitutionSpec;
+
+public class GelfOperation implements Operation {
+
+  private final FlattenOperation flattenOp;
+  private final SubstitutionOperation subOp;
+
+  private static final Set<String> GELF_FIELDS = new HashSet<>(Arrays.asList("version", "host",
```

Comment:
Move the list of strings down to the next line, so its not split up this funny way. :)

Reply:
Auto formatter formats to that.